Source/FieldSolver/FiniteDifferenceSolver/ApplySilverMuellerBoundary.cpp

```diff
@@ -244,18 +252,27 @@ void FiniteDifferenceSolver::ApplySilverMuellerBoundary (
                 // At the -z boundary (innermost guard cell)
                 if ( apply_lo_z && ( j==domain_box.smallEnd(1)-1 ) )
                     Bx(i,j,k) = coef1_z * Bx(i,j,k) + coef2_z * Ey(i,j+1,k);
+#elif WARPX_DIM_1D_Z
+                // At the +z boundary (innermost guard cell)
+                if ( apply_hi_z && ( i==domain_box.bigEnd(1)+1 ) )
```

Comment:
Am I correct that this should be `bigEnd(0)`?
```suggestion
                if ( apply_hi_z && ( i==domain_box.bigEnd(0)+1 ) )
```

Reply:
You are right. Changed it